src/odemis/driver/tescan.py

```diff
@@ -1096,6 +1107,60 @@ def _pollXYZ(self):
         except Exception:
             logging.exception("Unexpected failure during XYZ polling")
 
+    def _checkPosition(self, orig_pos: Dict[str, float], move: Dict[str, float], rel: bool = False,
+                       timeout: float = STAGE_WAIT_TIMEOUT):
+        """
+        Checks and waits for the current stage position to report the requested stage position given by pos.
+        :param orig_pos: original stage position before the stage movement in absolute coordinates
+        :param move: requested stage position in relative or absolute coordinates
+        :param rel: By default False, requested stage position is in absolution position otherwise relative
```

Comment:
This function is *always* called with `rel=False`. => drop this argument, and indicate that it's in absolute coordinates all the time.
Also, change the name of `move` to `target_pos`. Then, you can drop the first 4 lines of the function.